RegalyticsRegulatoryArticle.cs

```diff
@@ -84,6 +90,9 @@ public class RegalyticsRegulatoryArticle : BaseData
         [JsonProperty(PropertyName = "agencies")]
         public List<string> Agencies { get; set; }
 
+        [JsonProperty(PropertyName = "sector_type")]
+        public List<string> Sector { get; set; }
```

Comment:
The sector_type is a list of key-value pair (dictionary):
```
"sector_type": [{"name": "Financial"}, {"name": "Real Estate and Construction"}],
```

```csharp
public List<Dictionary<string,string>> Sector { get; set; }
```

Reply:
Sounds good, just fixed the issues. Thanks for the links they were very helpful